apps/aevm/src/aevm_eeevm.erl

```diff
@@ -1164,7 +1136,7 @@ loop(StateIn) ->
                     %% almost entirely identical.
                     {Res, State1} = recursive_call(State0, OP),
                     State2 = push(Res, State1),
-                    next_instruction(OP, State, State2);
+                    next_instruction(CP, State, State2);
                 16#f5 -> eval_error({illegal_instruction, OP}, State0);
```

Comment:
In eval_error cases, it seems the CP in the state will not be update hence [test log](https://github.com/aeternity/epoch/blob/297b095df5b35548b8bec4135aed2903136a38ce/apps/aevm/src/aevm_eeevm.erl#L73) may be misleading. Though it seems only a test issue.